src/main/java/dk/kb/storage/storage/DsStorage.java

```diff
@@ -792,37 +896,49 @@ public void updateRecord(DsRecordDto record) throws Exception {
     }
 
 
-    public void updateKalturaInternal(String kalturaReferenceId, String kalturaId) throws Exception {
+    public void updateKalturaIdForRecord(String kalturaReferenceId, String kalturaId) throws Exception {
       
         long nowStamp = UniqueTimestampGenerator.next();      
-        try (PreparedStatement stmt = connection.prepareStatement(updateKalturaReferenceIdStatement);) {
+        try (PreparedStatement stmt = connection.prepareStatement(updateKalturaIdStatement);) {
             stmt.setString(1, kalturaId);
             stmt.setLong(2, nowStamp);
             stmt.setString(3, kalturaReferenceId);  
             stmt.executeUpdate();
         } catch (SQLException e) {
-            String message = "SQL Exception in updateKalturaInternal for kalturaReferenceId:" + kalturaReferenceId + " error:" + e.getMessage();
+            String message = "SQL Exception in updateKalturaId for referenceId:" + kalturaReferenceId + " error:" + e.getMessage();
             log.error(message);
             throw new SQLException(message, e);
         }
 
     }
     
     
+    private static MappingDto createMappingFromRS(ResultSet rs) throws SQLException {
+
+        String id = rs.getString(MAPPING_REFERENCE_ID_COLUMN);
+        String kalturaId = rs.getString(KALTURA_ID_COLUMN);
```

Comment:
The variables uesd here are content-wise correct. However, you've introduced a variable named `MAPPING_KALTURA_ID_COLUMN` which should be used instead of `KALTURA_ID_COLUMN` as that relates to the records table

Reply:
Correct, was a (harmless) mistake.